.circleci/config.yml

```
@@ -33,10 +33,10 @@ docker_build: &docker_build
   run:
     name: Build docker images
     command: |
-      apk add --no-cache docker-cli-buildx
-      docker context create my_context
-      docker buildx create --use my_context
-      docker buildx build --platform linux/arm64,linux/amd64 $DOCKER_PATH
+       apk add --no-cache docker-cli-buildx
+       docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
+       docker buildx create --name multiarch --driver docker-container --use
+       docker buildx build --platform linux/arm64,linux/amd64 $DOCKER_PATH
```

```suggestion
       apk add --no-cache docker-cli-buildx
       docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
       docker context create multiarch
       docker buildx create multiarch --driver docker-container --use
       docker buildx build --platform linux/arm64,linux/amd64 $DOCKER_PATH
```

MAYBE this one?? CI debugging is painful :(